src/api/getUser.js

```diff
@@ -0,0 +1,16 @@
+import { API_BASE_URL, API_SAMPLE_USER } from "../constants/apiConstants";
+
+const getUser = async () => {
+  try {
+    const response = await fetch(`${API_BASE_URL + API_SAMPLE_USER}`);
+    if (!response.ok) {
+      throw new Error("유저 데이터를 불러오는데 실패했습니다.");
+    }
+    const body = await response.json();
+    return body;
```

Comment:
중간 변수가 꼭 있어야할 이유가 없다면 생략하는 것이 좋겠습니다!

```suggestion
    return response.json();
```